src/components/Header/Header.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[];
+}
+
+export const Header: React.FC<Props> = ({ todos }) => {
+  return (
+    <header className="todoapp__header">
+      {todos.length > 0 && (
+        // eslint-disable-next-line jsx-a11y/control-has-associated-label
+        <button type="button" className="todoapp__toggle-all active" />
```

Comment:
Instead of disabling eslint, add an appropriate aria-label attribute to the button for better accessibility.